ui/job-view/headerbars/ReposMenu.jsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { getRepoUrl } from '../../helpers/url';
-import { thRepoGroupOrder } from '../../js/constants';
+import { thRepoGroupOrder } from '../../helpers/constants';
```

Comment:
`thRepoGroupOrder` is only used in one place (and a place that's already been converted to React) - should we move the constant there?

Reply:
Good call, yeah.  :)